<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>udg::QDefaultViewerExtensionBase</class>
 <widget class="QWidget" name="udg::QDefaultViewerExtensionBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>529</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>WW/WL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_windowLevelComboBox" >
       <item>
        <property name="text" >
         <string>Default</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>CT Bone</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>CT Lung</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>CT Soft Tissues, noncontrast</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>CT Liver, noncontrast</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>CT Soft Tissues , Contrast Medium</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>CT Liver , Contrast Medium</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>CT Neck , Constrast Medium</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>CT Angiography</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Osteoporosis</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Emphysema</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Petrous Bone</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Custom...</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_axialViewToolButton" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_sagitalViewToolButton" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_coronalViewToolButton" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_switchViewToolButton" >
       <property name="text" >
        <string>Switch Double View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_chooseSeriePushButton" >
       <property name="text" >
        <string>Choose Serie To Compare</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_synchroCheckBox" >
       <property name="text" >
        <string>Synchronize Slices</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="m_stackedWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="udg::Q2DViewer" name="m_2DView" />
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="m_viewText" >
           <property name="text" >
            <string>XY : Axial</string>
           </property>
           <property name="scaledContents" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="m_slider" >
           <property name="focusPolicy" >
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="maximum" >
            <number>0</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="m_spinBox" >
           <property name="maximum" >
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="widget" >
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="udg::Q2DViewer" name="m_2DView2_1" />
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="m_viewText2_1" >
               <property name="text" >
                <string>XY : Axial</string>
               </property>
               <property name="scaledContents" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="m_slider2_1" >
               <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="maximum" >
                <number>0</number>
               </property>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="m_spinBox2_1" >
               <property name="maximum" >
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget" >
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="udg::Q2DViewer" name="m_2DView2_2" />
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="m_viewText2_2" >
               <property name="text" >
                <string>XY : Axial</string>
               </property>
               <property name="scaledContents" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="m_slider2_2" >
               <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="maximum" >
                <number>0</number>
               </property>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="m_spinBox2_2" >
               <property name="maximum" >
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>udg::Q2DViewer</class>
   <extends></extends>
   <header>q2dviewer.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
